src/team.coffee

```diff
@@ -22,196 +22,156 @@
+              else
+                responseMessage.memberAlreadyOutOfTeam(user, team)
+    msg.send message
 
   ##
   ## hubot <team_name> team -1 - remove me from the team
   ##
   robot.respond /(\S*)? team -1/i, (msg) ->
-    team_name = msg.match[1]
-    removeUserFromTeam(msg.message.user.name, team_name, msg)
+    teamName  = msg.match[1]
+    team      = if teamName
+                  Team.find(teamName)
+                else
+                  Team.default()
+    return msg.send responseMessage.teamNotFound(teamName) unless team
+    user = UserNormalizer.normalize(msg.message.user.name)
+    message = if team.removeMember user
+                responseMessage.memberRemovedFromTeam(user, team)
+              else
+                responseMessage.memberAlreadyOutOfTeam(user, team)
+    msg.send message
 
   ##
   ## hubot <team_name> team count - list the current size of the team
   ##
   robot.respond /(\S*)? team count$/i, (msg) ->
-    team_name = msg.match[1] || config.default_team_label
-    return unless teamExists(team_name)
-    msg.send "#{teamSize(team_name, msg)} people are currently in the team"
+    teamName  = msg.match[1]
+    team      = if teamName
+                  Team.find(teamName)
+                else
+                  Team.default()
+    message = if team
+                responseMessage.teamCount(team)
+              else
+                responseMessage.teamNotFound(teamName)
+    msg.send message
 
   ##
   ## hubot <team_name> team (list|show) - list the people in the team
   ##
   robot.respond /(\S*)? team (list|show)$/i, (msg) ->
-    team_name = msg.match[1] || config.default_team_label
-    return unless teamExists(team_name)
-    count = teamSize(team_name, msg)
-    if count == 0
-      msg.send "There is no one in the #{teamLabel(team_name)} currently"
-    else
-      position = 0
-      message = "#{teamLabel(team_name)} (#{count} total):\n"
-      for user in robot.brain.data.teams[team_name]
-        position += 1
-        message += "#{position}. #{user}\n"
-      msg.send message
+    teamName  = msg.match[1]
+    team      = if teamName
+                  Team.find teamName
+                else
+                  Team.default()
+    message = if team
+                responseMessage.listTeam(team)
+              else
+                responseMessage.teamNotFound(teamName)
+    msg.send message
 
   ##
   ## hubot <team_name> team (empty|clear) - clear team list
   ##
   robot.respond /(\S*)? team (clear|empty)$/i, (msg) ->
-    team_name = msg.match[1] || config.default_team_label
-    if msg.message.user.name in admins
-      robot.brain.data.teams[team_name] = []
-      msg.send "#{teamLabel(team_name)} list cleared"
-    else
-      msg.reply "Sorry, only admins can clear the team members list"
+    teamName  = msg.match[1]
+    team      = if teamName
+                  Team.find teamName
+                else
+                  Team.default()
```

Comment:
Same comment as [earlier](https://github.com/hubot-scripts/hubot-team/pull/7/files#r26907801)

Maybe just make the check a helper function `getTeam` so that you don't repeat it every time.